### PR TITLE
[WIP] Bootstrap4

### PR DESCRIPTION
**Pull request by andreadelrio**
I'm adding Bootstrap 4 to take advantage of its grid system (among other things, like forms). I'm using something called [reactstrap](https://reactstrap.github.io/) which has been very convenient so far.

I'm refactoring a lot of our .scss files and the file structure.

### CODE REVIEW

File: src/UniversalStyles.scss [hunk 1]
```diff
@@ -1,2 +1,4 @@
+@import 'https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0-alpha.4/css/bootstrap.min.css';
+// @import 'https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0-alpha.5/css/bootstrap-grid.min.css';
```
Comment (davidascher) [inline src/UniversalStyles.scss]: In general, I don't think we should commit commented-out code.  Just make the changes you think are needed.

File: src/components/AccordionTab.scss [hunk 2]
```diff
@@ -1,70 +1,70 @@
-@import "../UniversalStyles";
+// @import "../UniversalStyles";
```
Comment (davidascher) [inline src/components/AccordionTab.scss]: Here (and everywhere else) -- get rid of commented out code.

File: src/components/CaseEditor.js [hunk 3]
```diff
@@ -3,7 +3,9 @@ import React, {Component} from 'react'
 import {injectIntl, intlShape} from 'react-intl'
 import { Field } from 'redux-form'
 
+import '../containers/Case/Case.css'
```
Comment (davidascher) [inline src/components/CaseEditor.js]: This seems wrong.  I suspect the CSS needs to be in `components`, not in `containers`.

File: src/containers/Method.js [hunk 4]
```diff
@@ -1,179 +1,179 @@
...
-import CountryMap from '../components/CountryMap'
-import ContentPencil from 'material-ui/svg-icons/image/edit'
-import api from '../utils/api'
-import caseIconBookmark from '../img/pp-case-icon-bookmark.png'
-import caseIconSettings from '../img/pp-case-icon-settings.png'
-import caseIconFB from '../img/pp-case-icon-fb.png'
-import caseIconTW from '../img/pp-case-icon-tw.png'
-import caseIconShare from '../img/pp-case-icon-share.png'
-import moment from 'moment'
+// import React from 'react'
```
Comment (davidascher) [inline src/containers/Method.js]: just get rid of the file =)

File: src/components/SearchHitCategory/SearchHitCategory.js [hunk 5]
```diff
@@ -10,7 +10,7 @@ class SearchHitCategory extends React.Component {
           <div className='category-title'>
             {this.props.title}
           </div>
-          <div className='results'>
+          <div className='row results'>
```
Comment (andreadelrio) [inline src/components/SearchHitCategory/SearchHitCategory.js]: I can confirm that `results` does get applied.

File: src/containers/Method.js (hunk 4 above)
Comment (andreadelrio) [inline src/containers/Method.js]: Didn't want to get rid of file, just forgot to uncomment it. Adding it.

**Comment by andreadelrio** [comment]
@davidascher I made the changes, please merge when you get a chance. I prefer to have the reviewer do the merging.

**Pull request merged**